src/renderers/WebGLRenderer.js

```diff
@@ -2488,7 +2538,15 @@ class WebGLRenderer {
 
 		this.initTexture = function ( texture ) {
 
-			if ( texture.isCubeTexture ) {
+			if ( texture.isRenderTarget ) {
```

Comment:
The thing is `initTexture()` is not supposed to called with an instance of render target so I'm not sure about this change. You have mentioned this in https://github.com/mrdoob/three.js/issues/28282#issuecomment-2094832435 but we have a long-standing policy to _not_ use render target as textures and I don't think this is the right opportunity to change this policy. Hence, I also think `copyTextureToTexture()` should not used with render targets but with their texture property.

How about we add `initRenderTarget()` instead? Devs can then call this method in their app before using `copyTextureToTexture()` with an uninitialized render target.

Reply:
> How about we add WebGLRenderer.initRenderTarget() instead? 

This is fine with me - it still leads to a confusing situation where every other kind of texture can be initialized in `copyTextureToTexture` while render target textures specifically cannot. But understood that there would be larger architectural changes needed to support this correctly.

I can make the change once #28281 is merged.